testsuite/tests/singlecluster/conftest.py

```diff
@@ -34,10 +34,12 @@ def authorization_name(blame):
 
 
 @pytest.fixture(scope="module")
-def authorization(kuadrant, route, authorization_name, cluster, label):
+def authorization(request, kuadrant, route, gateway, blame, cluster, label):  # pylint: disable=unused-argument
     """Authorization object (In case of Kuadrant AuthPolicy)"""
+    target_ref = request.getfixturevalue(getattr(request, "param", "route"))
```

Comment:
I do not like that indirect parametrization is so high in the hierarchy. It probably wont affect anything, but I still do not like it. @jsmolar thoughts?

Reply:
I added exactly the same parametrization to rate limit tests and it is opaque shit. I don't know how to solve parametrization for routes and gateways, so if you have better idea I'm all ears, but if not lets go with this solution.